src/pages/Index.vue

```diff
@@ -363,11 +363,20 @@ export default {
     mounted() {
         this.setPrioritizedGalaxyLocales();
     },
+    metaInfo() {
+        return {
+            title: this.$page.main.title,
```

Comment:
It's much cleaner and more straightforward to just return `Home` here directly, skip the .md and page query.  There's just not a good reason for the obfuscation.